Makefile

```diff
@@ -11,80 +11,65 @@ all: assert-dom0
 	@echo
 	@echo "make dev"
 	@echo "make staging"
-	@echo "make prod"
 	@echo
 	@echo "These targets will set your config.json to the appropriate environment."
+	@false
 
-dev: assert-dom0 ## Configures and builds a DEVELOPMENT install
-	./scripts/configure-environment --env dev
+dev staging: assert-dom0 ## Configures and builds a dev or staging environment
+	./scripts/configure-environment --env $@
 	$(MAKE) validate
-	$(MAKE) prep-salt
-	./scripts/provision-all
-
-prod: assert-dom0 ## Configures and builds a PRODUCTION install for pilot use
-	./scripts/configure-environment --env prod
-	$(MAKE) validate
-	$(MAKE) prep-salt
-	./scripts/provision-all
-
-staging: assert-dom0 ## Configures and builds a STAGING install. To be used on test hardware ONLY
-	./scripts/configure-environment --env staging
-	$(MAKE) validate
-	$(MAKE) prep-salt
-	./scripts/provision-all
+	$(MAKE) prep-dev
+	sdw-admin --apply
 
 dom0-rpm: ## Builds rpm package to be installed on dom0
 	@./scripts/build-dom0-rpm
 
 clone: assert-dom0 ## Pulls the latest repo from work VM to dom0
 	@./scripts/clone-to-dom0
 
-qubes-rpc: prep-salt ## Places default deny qubes-rpc policies for sd-app and sd-gpg
+qubes-rpc: prep-dev ## Places default deny qubes-rpc policies for sd-app and sd-gpg
 	sudo qubesctl --show-output --targets sd-dom0-qvm-rpc state.highstate
 
 add-usb-autoattach: prep-dom0 ## Adds udev rules and scripts to sys-usb
 	sudo qubesctl --show-output --skip-dom0 --targets sys-usb state.highstate
 
-remove-usb-autoattach: prep-salt ## Removes udev rules and scripts from sys-usb
+remove-usb-autoattach: prep-dev ## Removes udev rules and scripts from sys-usb
 	sudo qubesctl --show-output state.sls sd-usb-autoattach-remove
 
-sd-workstation-template: prep-salt ## Provisions base template for SDW AppVMs
+sd-workstation-template: prep-dev ## Provisions base template for SDW AppVMs
 	sudo qubesctl --show-output state.sls sd-workstation-buster-template
 	sudo qubesctl --show-output --skip-dom0 --targets sd-workstation-buster-template state.highstate
 
-sd-proxy: prep-salt ## Provisions SD Proxy VM
+sd-proxy: prep-dev ## Provisions SD Proxy VM
```

Comment:
The changes to the dev environment may have repercussions on these individual makefile targets :

`prep-salt` makefile target would copy the local salt file in `/srv/salt/` whereas this updated `prep-dev` target will install the dom0 rpm. This means that any changes to the local files in the `securedrop-workstation` folder in dom0 will not be used. 

If this is the case, adding a note to this effect in the dev docs could be helpful

Reply:
> If this is the case, adding a note to this effect in the dev docs could be helpful

That's definitely true. Editing the files in e.g. /srv/salt/ still works fine, but I wouldn't recommend either, given how easy it is to lose changes that way. Will clarify in the docs!